subsys/bluetooth/shell/bt.c

```diff
@@ -2845,10 +2843,8 @@ static int cmd_connect_le(const struct shell *sh, size_t argc, char *argv[])
 					BT_GAP_SCAN_FAST_INTERVAL,
 					BT_GAP_SCAN_FAST_INTERVAL);
 
-	err = bt_conn_le_create(
-		&addr, create_params,
-		BT_LE_CONN_PARAM(BT_GAP_INIT_CONN_INT_MIN, BT_GAP_INIT_CONN_INT_MIN, 0, 400),
-		&conn);
+	err = bt_conn_le_create(&addr, create_params, BT_LE_CONN_PARAM_DEFAULT,
+				&conn);
```

Comment:
Seems like it'd make sense to update `bt_conn_le_create()` to accept `NULL` as the last parameter in cases like this (i.e. where you don't do anything meaningful with the pointer even after a successful return).